migration.sql

```diff
@@ -1,36 +1,23 @@
-CREATE TABLE restaurants (
-    id UUID PRIMARY KEY,
-    created_at TIMESTAMP,
-    updated_at TIMESTAMP,
-    deleted_at TIMESTAMP,
-    name VARCHAR(255),
-    avatar VARCHAR(255),
-    phone VARCHAR(200),
-    countryCode VARCHAR(5),
-    slug VARCHAR(255),
-    street VARCHAR(255),
-    district VARCHAR(255),
-    city VARCHAR(255),
-    full_address VARCHAR(255),
-    rating FLOAT,
-    username VARCHAR(255),
-    lat FLOAT,
-    long FLOAT,
-    is_opening BOOLEAN,
-    is_opening_24h BOOLEAN,
-    minutes_until_next_status INT,
-    promoted_at TIMESTAMP,
-    is_loship_partner BOOLEAN,
-    is_honored BOOLEAN,
-    quote TEXT,
-    is_active BOOLEAN,
-    is_checked_in BOOLEAN,
-    closed BOOLEAN,
-    recommended_ratio FLOAT,
-    recommended_enable BOOLEAN,
-    distance INT,
-    is_purchased_supply_items BOOLEAN,
-    is_sponsored BOOLEAN,
-    free_shipping_milestone INT
-);
-
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+create table restaurants
+(
+    id             uuid primary key DEFAULT uuid_generate_v1(),
+    created_at     timestamp with time zone,
+    updated_at     timestamp with time zone,
+    deleted_at     timestamp with time zone,
+    name           varchar,
+    phone          varchar,
+    slug           varchar,
+    street         varchar,
+    district       varchar,
+    city           varchar,
+    full_address   varchar,
+    lat            double precision,
+    long           double precision,
+    is_opening     boolean,
```

Comment:
our service is not support real-time, so I think we can remove it
```suggestion

```

Reply:
thanks, it's done